content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Pod Topology Spread Constraints
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+You can use _topology spread constraints_ to control how {{< glossary_tooltip text="Pods" term_id="Pod" >}} are spread across your cluster among failure-domains such as regions, zones, nodes, and other user-defined topology domains. This can help to achieve high availability as well as efficient resource utilization.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Motivation
+
+In Kubernetes, directives related to "Affinity" control how Pods are
+scheduled - more packed or more scattered. But right now only limited options
```

Comment:
As soon as this is merged into the _master_ branch, “right now” will be incorrect: there will be a new (alpha) feature that provides an alternative.

Reply:
I recommend moving this Motivation section further down the page, or even omitting it and putting a link into the What's Next section so that interested readers can find and read the KEP.

Reply:
How about moving it down the page, and rename to "Comparison with PodAffinity/PodAntiAffinity".